apps/examples/testcase/le_tc/compression/compression_tc_main.c

```diff
@@ -43,7 +44,10 @@ int tc_compression_main(int argc, char *argv[])
 	g_tc_fd = open(OS_API_TEST_DRVPATH, O_WRONLY);
 
 	/*If FAIL : Failed to open testcase driver*/
-	TC_ASSERT_GEQ("open", g_tc_fd, 0);
+	if (g_tc_fd < 0) {
```

Comment:
Why did you change like this?
All TCs check the failure with macro as previous.

Reply:
no. 
please check network_tc_main.c

Reply:
this is the error while building for artik053/tc

```
In file included from le_tc/compression/compression_tc_main.c:27:
le_tc/compression/compression_tc_main.c: In function 'tc_compression_main':
./tc_common.h:108:3: error: 'return' with no value, in function returning non-void [-Werror=return-type]
  108 |   return; \
      |   ^~~~~~
./tc_common.h:112:43: note: in expansion of macro 'TC_ASSERT_GEQ_CLEANUP'
  112 | #define TC_ASSERT_GEQ(api_name, var, ref) TC_ASSERT_GEQ_CLEANUP(api_name, var, ref, ((void)0))
      |                                           ^~~~~~~~~~~~~~~~~~~~~
le_tc/compression/compression_tc_main.c:46:2: note: in expansion of macro 'TC_ASSERT_GEQ'
   46 |  TC_ASSERT_GEQ("open", g_tc_fd, 0);
      |  ^~~~~~~~~~~~~
le_tc/compression/compression_tc_main.c:40:5: note: declared here
   40 | int tc_compression_main(int argc, char *argv[])
      |     ^~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[2]: *** [Makefile:118: compression_tc_main.o] Error 1

```

Reply:
Please add commit description about what build error is and what the reason is with build error logs.

Reply:
done